__tests__/index.tests.js

```diff
@@ -67,3 +67,40 @@ describe.only("GET /api", () => {
       });
   });
 });
+
+describe("GET /api/articles/:article_id", () => {
+  test("GET: 200 gets an article by its id with the correct properties", () => {
+    return request(app)
+      .get("/api/articles/3")
+      .expect(200)
+      .then(({ body }) => {
+        const { article } = body;
+
+        expect(article).toHaveProperty("author", "icellusedkars");
+        expect(article).toHaveProperty(
+          "title",
+          "Eight pug gifs that remind me of mitch"
+        );
+        expect(article).toHaveProperty("body", "some gifs");
+        expect(article).toHaveProperty("article_id", 3);
+        expect(article).toHaveProperty("topic", "mitch");
+        expect(article).toHaveProperty(
+          "created_at",
+          "2020-11-03T09:12:00.000Z"
+        );
+        expect(article).toHaveProperty("votes", 0);
+        expect(article).toHaveProperty(
+          "article_img_url",
+          "https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700"
+        );
+      });
+  });
+  test("GET: 404 article id does not exist and returns an error message", () => {
+    return request(app)
+      .get("/api/articles/459")
+      .expect(404)
+      .then((response) => {
+        expect(response.body.msg).toBe("Bad request");
```

Comment:
This is not a 'bad request' here. It's rather 'Not found'. Please change the message.